solidearth_code/solid_grid.f

```diff
@@ -0,0 +1,1613 @@
+      subroutine solidwrapped(iyr,imo,idy,ihh,imm,iss,glad1,glod1,
+     *                        glad2,glod2,steplon,steplat,increm,myfile)
+
+*** will compute 1 year of tide at given location starting at date hour 

```

Comment:
Hi @sssangha, this is awesome! I am trying to understand the usage of this function: the output text file is the predicted solid earth tide for the given date/time, right? This comment is not in the original *.for file from Dennis. And I got confused a little bit.

Reply:
Hello @yunjunz, yes this file has been modified from the original source code. Originally it generated solid earth tide predictions for a given time and point location. I modified it to generate a coarse grid of predictions for a given time, which I then interpolate to the interferometric grid. 
Since I am concerned though that the source code will diverge, I aim to instead transition to wrapping around the GMT solid earth tide function (which uses the same source code). In this way, we may directly generate grids and be tied to GMT instead of diverging.